karma.conf.js

```diff
@@ -17,8 +17,8 @@ module.exports = function (config) {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
-    autoWatch: true,
-    browsers: ['Chrome'],
+    autoWatch: false,
+    browsers: ['ChromeHeadlessNoSandbox'],
```

Comment:
Ok, this is good for circleci, but not good for development. Can you make a separete `karma.circle.conf` file with these settings?

Reply:
Ok!

Reply:
I added a new karma file @jellegerbrandy